client/my-sites/upgrades/navigation.jsx

```diff
@@ -123,14 +117,12 @@ var UpgradesNavigation = React.createClass( {
 		var items;
 
 		if ( this.props.selectedSite.jetpack ) {
-			items = [ 'Plans', 'My Upgrades' ];
-		} else if ( config.isEnabled( 'upgrades/domain-management/list' ) ) {
-			items = [ 'Plans', 'Domains', 'Email', 'My Upgrades' ];
+			items = [ 'Plans', 'My Purchases' ];
```

Comment:
Shouldn't we use `Manage Purchases` name as in `/me` sidebar?

![screen shot 2015-11-30 at 08 53 02](https://cloud.githubusercontent.com/assets/699132/11466261/ee71bf36-973f-11e5-89ea-64abc34d7741.png)